### PR TITLE
docs: update pull_request.md and PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
We have change the branch model.

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Each pull request should address a single issue and have a meaningful title.
 
+- All bug fixes should be sent to the __"develop"__ branch, this is where the next bug fix version will be developed.
+- PRs with any enhancement should be sent to __"4.3"__ branch, this is where the next minor upgrade version will be developed.
```
Comment (MGatner) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Can we make this generic so we don't have to keep updating the document?
```suggestion
- PRs with any enhancement should be sent to the next minor version branch, e.g. __"4.3"__
```

File: contributing/pull_request.md [hunk 2]
```diff
@@ -38,16 +38,20 @@ This might sound a bit tough, but it is required for us to maintain the quality
...
 
-	composer cs-fix
+```
+> composer cs-fix
+```
 
 You can check the coding style violations:
 
-	composer cs
+```
```
Comment (MGatner) [inline contributing/pull_request.md:52]: Specify `bash` formatting?

File: contributing/pull_request.md [hunk 3]
```diff
@@ -154,8 +158,9 @@ working on your contribution.
 
 ### Branching
 
-CodeIgniter4 uses the [Git-Flow](http://nvie.com/posts/a-successful-git-branching-model/) branching model
-which requires all Pull Requests to be sent to the __"develop"__ branch; this is where the next planned version will be developed.
+All bug fixes should be sent to the __"develop"__ branch, this is where the next bug fix version will be developed.
+
+PRs with any enhancement should be sent to __"4.3"__ branch, this is where the next minor upgrade version will be developed.
```
Comment (MGatner) [inline contributing/pull_request.md:163]:
```suggestion
PRs with any enhancement should be sent to next minor version branch, e.g. __"4.3"__
```

**Review by MGatner — CHANGES_REQUESTED** [review]

File: contributing/pull_request.md (hunk 2 above)
Comment (kenjis) [reply contributing/pull_request.md]: Added `console`.

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
https://github.com/github/linguist/blob/d7799da826e01acdb8f84694d33116dccaabe9c2/lib/linguist/languages.yml#L6063

Comment (MGatner) [reply contributing/pull_request.md]: Wasn't aware of "ShellSession", very cool.

**Review by MGatner — APPROVED** [review]

**Pull request merged**